<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QtVisionLab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="textConsole_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessed_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginal_3">
    <property name="geometry">
     <rect>
      <x>1509</x>
      <y>30</y>
      <width>151</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <widget class="QMenu" name="menuWebcam">
     <property name="title">
      <string>Webcam</string>
     </property>
     <addaction name="actionAdd_URL"/>
    </widget>
    <addaction name="actionExit"/>
    <addaction name="actionFile"/>
    <addaction name="menuWebcam"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ComputerVision</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionAdd_URL">
   <property name="text">
    <string>Add Url</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
